webp.cc

```diff
@@ -5,14 +5,17 @@
 #include "cairo.h"
 #include <stdint.h>
 
+using v8::Object;
+using v8::Local;
+

```

Comment:
Looks like these are unused.

Reply:
Interesting, when I try to compile without them I get errors like
```
‘Object’ was not declared in this scope; did you mean ‘v8::Object’?
```
which is why I originally added them in. Is this something related to the node version I'm on?

Reply:
Ah I see, I was only looking at the lines changed in this PR. Line 47L/50R uses `Local` and `Object`. 👍